src/test/ui/const-generics/incorrect-number-of-const-args.rs

```diff
@@ -6,6 +6,6 @@ fn foo<const X: usize, const Y: usize>() -> usize {
 }
 
 fn main() {
-    foo::<0>(); //~ ERROR wrong number of const arguments: expected 2, found 1
```

Comment:
I'm not sure what's going on here. Having too few type arguments still results in an error, and as far as I can tell, type arguments use the same code path as const arguments. It might be that a separate issue is suppressing this error, or I might simply have missed something.